openhantek/src/widgets/levelslider.cpp

```diff
@@ -539,8 +540,8 @@ QRect LevelSlider::calculateRect(int sliderId) {
 /// \brief Search for the widest slider element.
 /// \return The calculated width of the slider.
 int LevelSlider::calculateWidth() {
-    // At least 8 px for the needles
-    this->sliderWidth = 8;
+    // At least 12 px for the needles
+    this->sliderWidth = 12;
```

Comment:
Why do you need to set the `sliderWidth` again here? It is set in the constructor already. This should probably be a constexpr field.

Reply:
OK, makes sense. Will do.